pages/common/emulator.md

```diff
@@ -0,0 +1,36 @@
+# emulator
+
+> Manager Android emulators from the command line.
+> More information: <https://developer.android.com/studio/run/emulator-commandline>.
+
+- Display the help:
+
+`emulator -help`
+
+- Start an Android emulator device:
+
+`emulator -avd {{name}}`
+
+- Display the web cameras on your development computer that are available for emulation:
+
+`emulator -avd {{name}} -webcam-list`
+
+- Start an emulator overriding the facing back camera setting (or -camera-front for front camera):
```

Comment:
Perhaps this will read better?
```suggestion
- Start an emulator overriding the facing back camera setting (use `-camera-front` for front camera):
```